<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveSelection</class>
 <widget class="QDialog" name="RemoveSelection">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>18000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QWidget {
	background-color: rgb(191,193,194);
	border-color: black;
}
*/
/*	Push button*/
QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: lightgray;
	selection-background-color: gray;
}

QPushButton:hover {
	background-color: gray;
	border-radius: 3px;
}


/*Tool button*/
QToolButton {
	max-width: 20px;
	max-height: 20px;
	border: 0px solid gray;
}

/*Line edit*/
QLineEdit {
    border: 1px solid gray;
	border-radius: 3px;
    background-color: lightgray;
    selection-background-color: darkgray;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_pFrameTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>290</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="m_pLabelTitle">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Remove Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_pToolButtonCancel">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Resources/Icons/close-icon.png</normaloff>../../../../Resources/Icons/close-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pVerticalWidgets">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(191,193,194);
border-radius: 3px;</string>
        </property>
        <property name="text">
         <string>Would you like to remove the following items from the project:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="m_pListView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(191,193,194);
border-radius: 3px;
</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pFrameButtonBox">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_pPushButtonRemove">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pPushButtonCancel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: gray;
	selection-background-color: rgb(191,193,194);
}

QPushButton:hover {
	background-color:rgb(105,105,105);
	border-radius: 3px;
	color:white;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
